dotnet/src/SemanticKernel.Abstractions/Contents/ChatMessageContent.cs

```diff
@@ -76,13 +154,25 @@ public class ChatMessageContent : KernelContent
         : base(innerContent, modelId, metadata)
     {
         this.Role = role;
-        this.Encoding = encoding ?? Encoding.UTF8;
-        this.Items = items;
+        this._encoding = encoding ?? Encoding.UTF8;
+        this._items = items;
     }
 
     /// <inheritdoc/>
     public override string ToString()
     {
-        return this.Content ?? string.Empty;
+#pragma warning disable CA1031 // Do not catch general exception types
+        try
+        {
+            return this.Content ?? string.Empty;
+        }
+        catch
+        {
+            return string.Empty;
+        }
+#pragma warning restore CA1031 // Do not catch general exception types
```

Comment:
The change is no longer relevant and was necessary to prevent the ToString methods from throwing exceptions if the item with index 0 was not a text item. Since we moved away from the solution, I'm rolling back the change to the method.